scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/context/TaskContext.java

```diff
@@ -163,6 +164,10 @@ public String getScratchURI() {
         return scratchURI;
     }
 
```

Comment:
We should create a Builder for this class there is too many argument for the constructors. It is not maintainable.